### DISCUSSION TITLE
Fox Movies Croatia

### DISCUSSION THREAD

**Discussion by TurboPing** [iptv-org/community#890] [Q&A]
### Channel ID

FoxMovies.hr

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
According to our database this channel was closed on 1 October 2023.

https://iptv-org.github.io/channels/hr/FoxMovies
